staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -196,6 +196,18 @@ var (
 		[]string{"verb", "resource", "subresource"},
 	)
 
+	requestFilterLatency = compbasemetrics.NewHistogramVec(
+		&compbasemetrics.HistogramOpts{
+			Name: "apiserver_request_filter_duration",
+			Help: "How much time the request spends in apf",
+			Buckets: []float64{0.0, 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0, 10.0, 15.0, 20.0, 25.0, 30.0,
+				35.0, 40.0, 45.0, 50.0, 60.0, 70.0, 80.0, 90.0, 100.0, 125.0, 150.0, 175.0, 200.0, 300.0,
+				400.0, 500.0, 600.0, 700.0, 800.0, 900.0, 1000.0},
```

Comment:
The default settings will hold something for up to 15s.

How do you plan to represent requests that are held for the max time and then rejected?